### ISSUE TITLE
how should I change lr in detr if my training batch is 4?

### ISSUE THREAD

**Issue by formorebeat** [open-mmlab/mmdetection#4959]
I have 2 gpus and I set the training batchsize 4. The base config is under the 8*2 training. So how should I change the lr setting in AdamW optimizer?

**Comment by xvjiarui** [comment]
You may apply linear scale up rule. The adjusted learning rate should be `lr/8*GPUs`.

**Comment by CMobley7** [comment]
@xvjiarui , I thought the linear scale rule was `lr/8*GPUs`. Is it not?

**Comment by hhaAndroid** [comment]
@CMobley7  Yes
